packages/react/src/ButtonGroup/ButtonGroup.tsx

```diff
@@ -1,37 +1,52 @@
 import styled from 'styled-components'
 import {get} from '../constants'
-import sx from '../sx'
+import {type SxProp} from '../sx'
+import React from 'react'
+import Box from '../Box'
+import {defaultSxProp} from '../utils/defaultSxProp'
 import type {ComponentProps} from '../utils/types'
+import {useProvidedRefOrCreate} from '../hooks'
 
-const ButtonGroup = styled.div`
+export type ButtonGroupProps = React.PropsWithChildren<SxProp & ComponentProps<typeof StyledButtonGroup>>
+
+const StyledButtonGroup = styled.div<SxProp>`
   display: inline-flex;
   vertical-align: middle;
   isolation: isolate;
 
-  && > * {
+  && :is(button, a) {
```

Comment:
I don't actually think we need `:is` for this. I think we can just select `button` and `a` directly

```suggestion
  button,
  a {
```